widget/embedded/src/store/slices/wallets.ts

```diff
@@ -245,6 +250,76 @@ export const createWalletsSlice: StateCreator<
       });
     }
   },
+  fetchCustomTokensBalance: async (params) => {
+    const { tokens } = params;
+    const blockchainsAndAssets: { [key: string]: Asset[] } = {};
```

Comment:
The variable name could be more descriptive.  `tokensByBlockchain` would clearly indicate the variable's purpose.
```suggestion
    const tokensByBlockchain: { [key: string]: Asset[] } = {};
```